Branch-SDK/src/io/branch/indexing/BranchUniversalObject.java

```diff
@@ -315,6 +334,19 @@ public void userCompletedAction(String action, HashMap<String, String> metadata)
 
     /**
      * <p>
+     * Method to report user actions happened on this BUO. Use this method to report the user actions for analytics purpose.
+     * </p>
+     *
+     * @param action   A {@link BRANCH_STANDARD_EVENT }with value of user action name.  See {@link BRANCH_STANDARD_EVENT} for Branch defined user events.
+     * @param metadata A HashMap containing any additional metadata need to add to this user event
+     */
+    @SuppressWarnings("deprecation")
+    public void userCompletedAction(BRANCH_STANDARD_EVENT action, HashMap<String, String> metadata) {
```

Comment:
We are  still maintaining deprecated method
`public void userCompletedAction(String action, HashMap<String, String> metadata)`  
for backward compatibility. After a while we will make this method private and then remove deprecation